docs/releases.rst

```diff
@@ -36,6 +27,40 @@ contrast to the :ref:`/spec/core/contact` key, this field is not
 supposed to be updated and can be useful when trying to track down
 the original author for consultation.
 
+Add support for running playbooks from Ansible collections specified
+using the ``namespace.collection.playbook`` notation.
+
+Added ``--dry`` option for the ``beaker`` provision plugin. When
+used it prints the Beaker Job XML without submitting it.
+
+:ref:`Results specification documentation</spec/results>` has now
+a dedicated place in the specification for improved discoverability.
+
+The ``rpm-ostree`` package installation now includes the
+``--assumeyes`` option for improved compatibility.
+
+Verbosity levels in ``tmt * show`` commands are now honored.
+
+Added new traceback verbosity level, ``locals``. It stands between
```

Comment:
The `locals` value is implementation detail only and cannot be used from the command line. What about mentioning `TMT_SHOW_TRACEBACK=2` instead and pointing users to [command variables](https://tmt--3496.org.readthedocs.build/en/3496/overview.html#command-variables) for more details?

Reply:
Ha, I was still under the impression that `locals` is supported, but it's not, it's indeed `2`.

Reply:
:facepalm: